### ISSUE TITLE
Allow URIs to be set as executable

### ISSUE THREAD

**Issue by ssk2** [mesos/chronos#416]
It looks like URIs cannot be set as executable via a Chronos job definition (as far as I am aware, correct me if I'm wrong). If the command is an HTTP or FTP URI then it is set as executable.

Relevant code: https://github.com/mesos/chronos/blob/36a294d4714e545de91fcda920429faff15a1746/src/main/scala/org/apache/mesos/chronos/scheduler/mesos/MesosTaskBuilder.scala#L101

It should be possible to manually specify URIs to be executable, or, alternatively, just make every URI executable (since we'll be running inside a sandbox anyway).

**Comment by ssk2** [comment]
Mesos ticket created: https://issues.apache.org/jira/browse/MESOS-2586

**Comment by elingg** [comment]
Good point, I think we should go with making every URI executable. This should be safe for chronos to have executable access to its sandbox environment.
